### PR TITLE
Fix wrong version for operator SDK

### PR DESCRIPTION
**Pull request by FreeK**
## Additional Information
Stumbled across the wrong version number for the operator SDK.

## Verification Steps
Run `make setup/operator-sdk`

## Checklist:
- [ ] Verified by team member
- [ ] Comments where necessary
- [ ] Automated Tests
- [ ] Documentation changes if necessary

## Additional Notes

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -2,7 +2,7 @@
 NAMESPACE=keycloak
 PROJECT=keycloak-operator
 PKG=github.com/keycloak/keycloak-operator
-OPERATOR_SDK_VERSION=v0.12.1
+OPERATOR_SDK_VERSION=v0.15.2
```
Comment (slaskawi) [inline Makefile]: This actually should be `v0.15.1`, see https://github.com/keycloak/keycloak-operator/blob/master/go.mod#L9

Comment (FreeK) [reply Makefile]: Updated to 0.15.1

**Comment by slaskawi** [comment]
@FreeK We still have 2 commits instead of one :) Could you please squash it?

**Comment by FreeK** [comment]
> Let me squash it and merge

Ah, sorry. I had some issues with my mailserver and I don't check GitHub daily.
Thanks for the quick fix!

**Pull request merged**